stdpopsim/catalog/GasAcu/genome_data.py

```diff
@@ -1,30 +1,2 @@
 # File autogenerated from Ensembl REST API. Do not edit.
-data = {
-    "assembly_accession": "GCF_016920845.1",
-    "assembly_name": "GAculeatus_UGA_version5",
-    "chromosomes": {
-        "1": {"length": 29619991, "synonyms": []},
-        "2": {"length": 23686546, "synonyms": []},
-        "3": {"length": 17759012, "synonyms": []},
-        "4": {"length": 34181212, "synonyms": []},
-        "5": {"length": 15550311, "synonyms": []},
-        "6": {"length": 18825451, "synonyms": []},
-        "7": {"length": 30776923, "synonyms": []},
-        "8": {"length": 20553084, "synonyms": []},
-        "9": {"length": 20843631, "synonyms": []},
-        "10": {"length": 17985176, "synonyms": []},
-        "11": {"length": 17651971, "synonyms": []},
-        "12": {"length": 20694444, "synonyms": []},
-        "13": {"length": 20748428, "synonyms": []},
-        "14": {"length": 16147532, "synonyms": []},
-        "15": {"length": 17318724, "synonyms": []},
-        "16": {"length": 19507025, "synonyms": []},
-        "17": {"length": 20195758, "synonyms": []},
-        "18": {"length": 15939336, "synonyms": []},
-        "19": {"length": 20580295, "synonyms": []},
-        "20": {"length": 20445003, "synonyms": []},
-        "21": {"length": 17421465, "synonyms": []},
-        "Y": {"length": 15859692, "synonyms": []},
-        "MT": {"length": 16543, "synonyms": []},
-    },
-}
+data = {"assembly_accession": None, "assembly_name": "BROAD S1", "chromosomes": {}}
```

Comment:
uh-oh, what happened to this one?

Reply:
wow that's not good

Reply:
it looks to have all the info there: http://rest.ensembl.org/info/assembly/gasterosteus_aculeatus?synonyms=1